.github/workflows/typing_conformance_comment.yaml

```diff
@@ -61,7 +77,7 @@ jobs:
           # subsequent runs
           echo '<!-- generated-comment typing_conformance_diagnostics_diff -->' >> comment.txt
 
-          echo '## Diagnostic diff on typing conformance tests' >> comment.txt
+          echo "## Diagnostic diff on [typing conformance tests](https://github.com/python/typing/tree/${CONFORMANCE_SUITE_COMMIT}/conformance)" >> comment.txt
```

Comment:
nit: it might be worth linking directly to the `conformance/tests` directory?

Reply:
I considered it, but I like that this page has a `README.md` file (which might be useful for first-time contributors!), whereas the `tests` subdirectory doesn't :-)